lib/tasks/dev.rake

```diff
@@ -0,0 +1,48 @@
+namespace :server do
+  desc "Configures and starts dependent servers"
+  task initialize: :environment do
+    Rake::Task["db:create"].invoke
+    Rake::Task["db:migrate"].invoke
+  end
+
+  desc "Start solr and postgres servers using lando."
+  task start: :environment do
+    system("lando start")
+    system("rake server:initialize")
+    system("rake server:initialize RAILS_ENV=test")
+  end
+
+  desc "Stop lando solr and postgres servers."
+  task stop: :environment do
+    system("lando stop")
+  end
+
+  namespace :solr do
+    task start_solr_wrapper: :environment do
+      SolrWrapper.wrap(port: 8983, verbose: true, managed: true, download_dir: 'tmp', version: '8.4.1') do |solr|
```

Comment:
I can run this task locally only if I create the databases manually without using lando. The lando task to create the db in dev and test fails.